### DISCUSSION TITLE
This plugin is awesome

### DISCUSSION THREAD

**Discussion by biscotty666** [mdelobelle/metadatamenu#435] [General]
I just went through the new Properties plugin and it is much less powerful than your plugin. Please don't go away.

**Comment by mdelobelle** [comment]
Thanks a lot for your comment. My entire workflow depends heavily on it, so I guess I'll do my best to keep improving it.
From what the core team is explaining, I understand that the Property plugin is focused on UI improvement for frontmatter data. 
It is very opinionated on the way data is structured (e.g. lists as indented list only and not as array) and has no plan to bound properties values lists with notes properties/tags/folders whatsoever; and last they don't provide access to a note's property from many locations (not in the tab header, not in explorer's context menu as far as I can tell from the tests I've made during alpha test phase..)
That's why I think there is still some added value with metadata menu, and I'm glad others think it too, and I will maintain it as long as I can

**Comment by biscotty666** [comment]
I really appreciate:

- like you said, the icon allowing you to open the modal from pretty much anywhere
- the modal :-)
- ability to nest fileClasses
- ability to maintain multi-select values in a plain note
- "add missing fields" wherever I want in my note
- edit key definitions via the modal
- the right click menu functionality in general 
- full support for in-line, double colon syntax

The last is a big one. I use yaml happily when programming, but Obsidian's implementation caused problems every time I tried. And with the new icon view of frontmatter it isn't even nicely compact anymore.

I've only used your plugin a short time but I can't live without it already.

**Comment by SpocWeb** [comment]
Hi @mdelobelle thank you so much for this Plugin. This supports me so much in structuring data. 
I have actually generated the full Hierarchy of file-Classes from the [Schema.org](https://schema.org/docs/full.html) in my open-source Vault on Github: https://github.com/SpocWiki/_Standards/tree/main/fileClass
Anyone can clone this as a sub-vault into their own Vaults, so they can use an open-source Vocabulary for their data. 
I hope that this is useful to many people and would like to get some feedback.
